src/test/cli/rbd/help.t

```diff
@@ -144,7 +144,7 @@ Skip test on FreeBSD as it generates different output there.
     --io-type arg        IO type (read , write, or readwrite(rw))
   
   rbd help children
-  usage: rbd children [--pool <pool>] [--image <image>] [--snap <snap>] 
+  usage: rbd children [--pool <pool>] [--image <image>] [--snap <snap>] [--all] 
```

Comment:
These (related) changes seem to be mixing and matching between commits. The title of this commit is to fix a bug re: listing children for trashed images so it should be limited to handling that fix. The next commit can add the new API methods and expand the rbd CLI to list trashed images.